addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1428,16 +1428,21 @@ def channel_fetched(self):
             """, member_query))
             members = cursor_self.env['discuss.channel.member'].browse([r[0] for r in to_update])
             channel2message = members.channel_id._get_last_messages().grouped('channel_id')
+            members_to_broadcast = self.env["discuss.channel.member"]
             for member in members:
                 last_message = channel2message[member.channel_id]
                 if member.fetched_message_id != last_message:
                     member.fetched_message_id = last_message
-                    member.channel_id._bus_send('discuss.channel.member/fetched', {
-                        'channel_id': member.channel_id.id,
-                        'id': member.id,
-                        'last_message_id': last_message.id,
-                        'partner_id': cursor_self.env.user.partner_id.id,
-                    })
+                    members_to_broadcast |= member
+            for channel in members.channel_id:
+                Store(bus_channel=channel).add(
+                    channel.channel_member_ids & members_to_broadcast,
```

Comment:
`channel_member_ids` should almost never be used, we don't want to fetch all members here.